### ISSUE TITLE
nf_conntrack_helper file not found

### ISSUE THREAD

**Issue by equaeghe** [firewalld/firewalld#353]
I am using firewalld 0.5.1 on Gentoo. It get an error message:
```
ERROR: Failed to read file "/proc/sys/net/netfilter/nf_conntrack_helper": [Errno 2] No such file or directory: '/proc/sys/net/netfilter/nf_conntrack_helper' 
```
My assumption is that I am missing some kernel module. However, I have no clue which one, as the usual suspects (IPv4/6 connection tracking support) are present. Can you shed light on what module(s) I might be missing?

**Comment by erig0** [comment]
@equaeghe, that proc file should exist if module "nf_conntrack" is loaded. Can you try loading it manually? firewalld should cause it to be loaded.

**Comment by erig0** [comment]
In firewalld 0.6.0 this module is explicitly loaded at start. Previously it was implicitly loaded due to default rules added by firewalld. Perhaps you can try firewalld 0.6.0.

**Comment by equaeghe** [comment]
This issue was resolved with an update to 0.6.3. (Other issues arose, but I'll open a new report for those.)
